pages/episode/[episodeId].tsx

```diff
@@ -102,6 +108,13 @@ export default function Episode({
     }
   }
 
+  if (serverEpisode.funding && serverEpisode.podcast.funding) {
+    const combinedFundingLinks = serverEpisode.funding.concat(serverEpisode.podcast.funding)
+    fundingLinks = combinedFundingLinks.map((link, index) => {
+      return <FundingLink key={index} link={link.url} value={link.value}></FundingLink>
```

Comment:
You should generally avoid using indexes as keys, in this circumstance, I think `link.url` will be unique in this group and should be used as the key.